blog/2025/04-15-XboxUWP-0.9/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Jellyfin for Xbox 0.9.0
+authors:
+  - JPVenson
+slug: xbox-v0.9.0
+tags: [release, xbox, uwp]
+---
+
+Better late then never. The UWP app for Jellyfin now has a new Maintainer and its first release in 5 years.
+
+/- JPVenson
+
+{/* truncate */}
+
+The Jellyfin app on Xbox has been unmaintained for quite a while. 
+While it was on lifesupport for the last couple of years, we were occasionally merging PRs and fixing bugs.
+
+With the last release now 5 years old, and the continued interest in Jellyfin being available on Xbox overall, I decided to take over at least the superficial duties of keeping it operational and releasing if there are new Features or bug fixes.
+
+# UWP 0.9.0 release
+
+This new release has now been approved and is available as a new version on the Microsoft store for up-to-date Xbox consoles.
+
+It contains a number of bug fixes, but the main update is the move to WebView2. 
+As the app is nothing more then a web browser with a few tweaks, this is a huge step up and allows Xboxes to use a Chromium-based browser instead of the old Legacy-Edge one. 
```

Comment:
```suggestion
As the app is nothing more than a web browser with a few tweaks, this is a huge step up and allows the Xbox to use a Chromium-based browser instead of the old, legacy Edge one. 
```